core/cap-0023.md

```diff
@@ -344,6 +341,20 @@ The behavior of `CreateClaimableBalanceOp` is as follows:
     - `asset` as specified in the operation
     - `amount` as specified in the operation
     - `reserve` equal to `claimants.size() * baseReserve`
+
+[^id-arithmatic]: ```
+    OperationID = (
+      sourceAccount ||
+      seqNum ||
+      opNum
+    )
+    balanceID.v0() = hex(sha256(OperationID))
+    ```
+    - `OperationID`: a union within the new `type` `ENVELOPE_TYPE_OP_ID`
```

Comment:
`operationID` is a struct, not a union - https://github.com/stellar/stellar-xdr/blob/78ef9860777bd53e75a8ce8b978131cade26b321/Stellar-transaction.x#L681-L686, which the original CAP wasn't clear on.

Reply:
Newer to xdr and meant it in the cryptographic combination sense of a concatenation via `||`. Thanks for the clarification! Will change it here in the footnote.